src/app/modules/project/overview/overview.component.html

```diff
@@ -112,7 +112,7 @@ <h3>Filter</h3>
   <div class="col-md">
     <div class="search">
       <input type="search" placeholder="search project" [formControl]="searchControl">
-      <button><img alt="" src="assets/images/research.svg" loading="lazy"></button>
+      <button><img alt="search" src="assets/images/research.svg" loading="lazy"></button>
```

Comment:
I would suggest making all the alt sentences start with a capital.

Reply:
Didn't notice, thanks! Updated now